packages/truffle-config/index.js

```diff
@@ -7,8 +7,8 @@ const findUp = require("find-up");
 const originalrequire = require("original-require");
 const Configstore = require("configstore");
 
-const DEFAULT_CONFIG_FILENAME = "truffle.js";
-const BACKUP_CONFIG_FILENAME = "truffle-config.js"; // For Windows + Command Prompt
+const DEFAULT_CONFIG_FILENAME = "truffle-config.js";
+const BACKUP_CONFIG_FILENAME = "truffle.js"; // For Windows + Command Prompt
```

Comment:
comment is outdated, might want to update it to say "old config filename" or somesuch